environment/src/main/java/com/sequenceiq/environment/environment/domain/EnvironmentTags.java

```diff
@@ -38,4 +41,13 @@ public String toString() {
                 + ", defaultTags=" + defaultTags
                 + '}';
     }
+
+    static EnvironmentTags fromJson(Json tags) {
```

Comment:
why do you use static instead of constructor?

Reply:
There is no particular reason; that is how the logic got extracted by IntelliJ and I did not feel it beneficial turning it into a constructor. :smile: